site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,483 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I'm pleased to announce the release of intake-esm version 2020.3.16. This is a new release with bug fixes and new features. Everyone is invited to give it a try, and make their thoughts, suggestions, feedback known! This blogpost outlines these changes. Full changelog is available [here](https://intake-esm.readthedocs.io/en/latest/changelog.html#intake-esm-v2020-03-16).\n",
+    "\n",
+    "\n",
+    "On GitHub: https://github.com/NCAR/intake-esm\n",
+    "\n",
+    "Documentation: https://intake-esm.readthedocs.io/\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "Intake-esm can be installed from PyPI with pip:\n",
+    "\n",
+    "```bash\n",
+    "python -m pip intall intake-esm --upgrade\n",
+    "```\n",
+    "\n",
+    "It is also available from conda-forge channel for conda isntallations:\n",
+    "\n",
+    "```bash\n",
+    "conda install -c conda-forge intake-esm\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## New Features\n",
+    "\n",
+    "\n",
+    "### Enhanced search: enforce query criteria via `require_all_on` argument\n",
+    "\n",
+    "By default intake-esm's `search()` method returns entries that fulfill **any of the criteria** specified in the query. Today intake-esm can return entries that fulfill **all query criteria** when the user supplies the `require_all_on` argument. `require_all_on` consits of attributes/dataframe columns to use when enforcing the query criteria.\n",
+    "\n",
+    "`require_all_on argument` is best explained with examples:\n"
```

Comment:
This line has the back-tick in the wrong place, and I would probably change it to:

```suggestion
    "The `require_all_on` argument is best explained with the following example.  Consider the `intake-esm` catalog for the CMIP6 data stored on Google Cloud Storage:\n"
```